mystyle.css

```diff
@@ -523,6 +531,54 @@ textarea {
   .set-image-desk {
     display: none;
   }
+
+  .drop-menu {
+    background-color: #3c3a39;
+    height: 100vh;
+    display: none;
+  }
+
+  .drop-menu.active {
+    display: block;
+  }
+
+  .menu-button.active .hamburger:nth-child(2) {
+    opacity: 0;
+  }
+
+  .menu-button.active .hamburger:nth-child(1) {
+    transform: translateY(7px) rotate(45deg);
```

Comment:
- [ ] Kindly make sure that mobile menu options follow the text colour as per the Figma design with is #FFF5E1.

Your design: 
<img width="501" alt="Screenshot 2022-04-06 at 2 38 31 AM" src="https://user-images.githubusercontent.com/70269507/161868539-bb68a245-52a6-41b3-b766-ac1c43ce0e0e.png">


Figma design:
<img width="342" alt="Screenshot 2022-04-06 at 2 38 51 AM" src="https://user-images.githubusercontent.com/70269507/161868481-a3961763-c87a-41f3-855c-4d00024c4d3a.png">